src/WCSimDetectorConstruction.cc

```diff
@@ -654,6 +654,10 @@ WCSimWLSProperties *WCSimDetectorConstruction::CreateWLSObject(G4String WLSType)
     WCSimWLSProperties* WLS = new EljenEJ286;
```

Comment:
On second thought, why not move the `WLS_PVT` material definition and `WlsPlasticMPT` table from `src/WCSimConstructMaterials.cc` to this `if else`. Then one can just change the `WLSType` variable in `src/WCSimDetectorConfigs.cc`. 
Although one still needs to recompile to change the WLS option, at least FD and IWCD are decoupled.